tests/mongo.yml

```diff
@@ -1,97 +1,92 @@
-      - mongo/jstests/core/arrayfind5.js
+      - tests/mongo/jstests/core/arrayfind5.js
 
       # https://github.com/FerretDB/FerretDB/issues/161
-      - mongo/jstests/core/basic1.js
+      - tests/mongo/jstests/core/basic1.js
 
       # https://github.com/FerretDB/FerretDB/issues/1710
       # https://github.com/FerretDB/FerretDB/issues/731
-      - mongo/jstests/core/arrayfind3.js
-      - mongo/jstests/core/arrayfind6.js
-      - mongo/jstests/core/arrayfind7.js
-      - mongo/jstests/core/arrayfind8.js
-      - mongo/jstests/core/arrayfind9.js
+      - tests/mongo/jstests/core/arrayfind3.js
+      - tests/mongo/jstests/core/arrayfind6.js
+      - tests/mongo/jstests/core/arrayfind7.js
+      - tests/mongo/jstests/core/arrayfind8.js
+      - tests/mongo/jstests/core/arrayfind9.js
 
       # https://github.com/FerretDB/FerretDB/issues/1492
-      - mongo/jstests/core/auth1.js
+      - tests/mongo/jstests/core/auth1.js
 
       # https://github.com/FerretDB/FerretDB/issues/1750
       # logout deprecated since version 5.0.
-      - mongo/jstests/core/auth2.js
+      - tests/mongo/jstests/core/auth2.js
 
       # https://github.com/FerretDB/FerretDB/issues/1745
-      - mongo/jstests/core/find_and_modify.js
-      - mongo/jstests/core/find_and_modify2.js
-      - mongo/jstests/core/find_and_modify3.js
-      - mongo/jstests/core/find_and_modify4.js
+      - tests/mongo/jstests/core/find_and_modify.js
+      - tests/mongo/jstests/core/find_and_modify2.js
+      - tests/mongo/jstests/core/find_and_modify3.js
+      - tests/mongo/jstests/core/find_and_modify4.js
 
       # https://github.com/FerretDB/FerretDB/issues/1384
-      - mongo/jstests/core/uniqueness.js
+      - tests/mongo/jstests/core/uniqueness.js
 
       # https://github.com/FerretDB/FerretDB/issues/1242
-      - mongo/jstests/core/unset2.js
+      - tests/mongo/jstests/core/unset2.js
 
       # https://github.com/FerretDB/FerretDB/issues/503
-      - mongo/jstests/core/update8.js
-      - mongo/jstests/core/updatec.js
-      - mongo/jstests/core/updatea.js
-      - mongo/jstests/core/updatei.js
-      - mongo/jstests/core/updatej.js
+      - tests/mongo/jstests/core/update8.js
+      - tests/mongo/jstests/core/updatec.js
+      - tests/mongo/jstests/core/updatea.js
+      - tests/mongo/jstests/core/updatei.js
+      - tests/mongo/jstests/core/updatej.js
 
       # https://github.com/FerretDB/FerretDB/issues/503
       # https://github.com/FerretDB/FerretDB/issues/1736
-      - mongo/jstests/core/update9.js
+      - tests/mongo/jstests/core/update9.js
 
       # https://github.com/FerretDB/FerretDB/issues/1742
-      - mongo/jstests/core/updateb.js
-      - mongo/jstests/core/updatel.js
+      - tests/mongo/jstests/core/updateb.js
+      - tests/mongo/jstests/core/updatel.js
 
       # https://github.com/FerretDB/FerretDB/issues/1744
-      - mongo/jstests/core/updatee.js
-
-      # https://github.com/FerretDB/dance/issues/304
-      - mongo/jstests/core/updatef.js
```

Comment:
Has this been removed by accident? :thinking:

Reply:
updatef.js was removed because https://github.com/FerretDB/dance/issues/304 was fixed.